Changes to DownloadUrlResolver

anilgit90 opened the pull request:
1. Changed signature resolving.
2. Changed player regex url resolver.

anilgit90 committed:
Changes to DownloadUrlResolver
1. Changed signature resolving.
2. Changed player regex url resolver.

anilgit90 committed:
Added GUI Example
1. Updated ExampleApplication to support GUI example
2. Added the project in Main Project Solution.

anilgit90 committed:
Added Manifest file for each project.

anilgit90 committed:
Changes to Project Settings
1. Added Application Icon
2. Exported Project Dependencies to Common Folder
3. Initial Appveyor Script for CI.

anilgit90 committed:
Changes to Appeyor Configuration v1

anilgit90 committed:
Changes to Appeyor Configuration v2

anilgit90 committed:
Changes to Appeyor Configuration v3

anilgit90 committed:
Changes to Appeyor Configuration v4

anilgit90 committed:
Changes to Appeyor Configuration v5

anilgit90 committed:
Updated README to include Appveyor Badge and Sample GUI application.

ac-lap commented:
Nice job dude! Will try and code-review/test your changes sometime next week or post Christmas.
Given the main repo is not being actively maintained, let's see if we can maintain your forked repo.

anilgit90 committed:
Changes to Sample GUI application
1. Added Progress bar to update the download progress of the video.
2. Updated readme to include feature list of Sample GUI.

HelgeStiemerling commented:
Hello,

self problem with: https://www.youtube.com/watch?v=G4mlA65wsbE

  public static string DecipherWithVersion(string cipher, string cipherVersion)
        {
            string jsUrl = string.Format("http://s.ytimg.com/yts/jsbin/player-{0}.js", cipherVersion);
                      
            string js = HttpHelper.DownloadString(jsUrl);
            
404 file not found jsUrl

![ex](https://user-images.githubusercontent.com/5903368/50310739-3791e780-04a3-11e9-8fa8-b6702a96c295.png)

anilgit90 committed:
Changes to Decipherer and readme.
1. Changed Decipherer.cs to Include changes to Youtube Player Url.
2. Updated Readme to include badge in new line.

anilgit90 committed:
Updated Appveyor badge

anilgit90 committed:
Updated Appveyor Badge

anilgit90 committed:
Updated to Include BetterCode Badge.

Added Analysis of code by BetterCode.

anilgit90 committed:
Removed Duplicate.

rcabreu commented:
This did not download https://www.youtube.com/watch?v=7rwW1ErcVu8
